jellyfin-platform-android/build.gradle.kts

```diff
@@ -16,9 +16,6 @@ android {
 
 	compileOptions {
 		isCoreLibraryDesugaringEnabled = true
-
-		sourceCompatibility = JavaVersion.VERSION_1_8
-		targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Even if it's enabled by default, shouldn't those be kept anyway?

Reply:
No because they default ti Java 8 now

Reply:
I know, but when thinking about it yesterday when I read it in the blogpost, I thought it'd still make sense to keep those for future proofing. Then again, it'll probably stay on 8 for quite a while.